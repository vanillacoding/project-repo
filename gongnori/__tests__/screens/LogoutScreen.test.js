import React from "react";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react-native";
import LogoutScreen from "../../screens/LogoutScreen";

describe("<LogoutScreen />", () => {
  test("snapshot test", () => {
    const component = renderer.create(<LogoutScreen />);

    expect(component).toMatchSnapshot();
  });

  test("indicator text should be rendered", () => {
    const { queryByText } = render(<LogoutScreen />);

    expect(queryByText("Logout")).not.toBeNull();
  });
});
