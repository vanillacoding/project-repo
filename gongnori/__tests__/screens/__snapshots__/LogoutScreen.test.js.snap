// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoutScreen /> snapshot test 1`] = `
<View>
  <Text>
    Logout
  </Text>
</View>
`;
